packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -79,424 +130,549 @@
     </dd>
     <dt>icon <code>string</code></dt>
     <dd>
-      <p>Any valid flight-icon name. If none is defined,
-        <span class="dummy-code default">user</span>
-        is set by default</p>
+      <p>Acceptable value: any Flight icon name.</p>
+      <p>Default: <span class="default">"user"</span></p>
     </dd>
     <dt>hasChevron <code>boolean</code></dt>
     <dd>
-      <p>
-        Acceptable values:
-      </p>
-      <ol>
-        <li class="default">true</li>
-        <li>false</li>
-      </ol>
+      <p>Default: <span class="default">true</span></p>
       <p>Per design,
-        <span class="dummy-code">false</span>
+        <code class="dummy-code">false</code>
         is only acceptable when the "more-horizontal" icon is used; as such, it is set to
-        <span class="dummy-code">true</span>
+        <code class="dummy-code">true</code>
         by default.</p>
     </dd>
-    <dt>
-      imageSrc
-      <code>string</code>
-    </dt>
+    <dt>imageSrc <code>string</code></dt>
     <dd>
       The URL of an image to be used instead of an icon (e.g., as an avatar).
     </dd>
+    <dt>“splattributes”</dt>
```

Comment:
```suggestion
    <dt>...attributes</dt>
```

[issue] We use the technically accurate names elsewhere, it should be used here as well.

Reply:
No problem, I am going to replace it across the whole documentation (Component API).

In general, I've found this term used in a lot of places of the official Ember documentation. Even on some RFCs: https://github.com/emberjs/rfcs/blob/master/text/0435-modifier-splattributes.md

Reply:
https://github.com/hashicorp/design-system/pull/235/commits/169237bc6b833348b84a4ed2d684e4d4d97af209